pcsx2/SPR.cpp

```diff
@@ -179,14 +179,14 @@ int  _SPR0chain()
 
 __fi void SPR0chain()
 {
-	int cycles = _SPR0chain() * BIAS;
+	const int cycles = _SPR0chain() * BIAS;
 	CPU_INT(DMAC_FROM_SPR, cycles);
 }
 
 void _SPR0interleave()
 {
 	int qwc = spr0ch.qwc;
-	int sqwc = dmacRegs.sqwc.SQWC;
+	const int sqwc = dmacRegs.sqwc.SQWC;
```

Comment:
Can you not rearrange these to not be staggered? just looks ugly :P maybe put it 3rd?